src/BigInteger/AdapterPluginManager.php

```diff
@@ -21,13 +22,23 @@
 class AdapterPluginManager extends AbstractPluginManager
 {
     /**
-     * Default set of adapters
+     * Default set of adapter aliases
      *
-     * @var array
+     * @var string[]
      */
-    protected $invokableClasses = [
-        'bcmath' => 'Zend\Math\BigInteger\Adapter\Bcmath',
-        'gmp'    => 'Zend\Math\BigInteger\Adapter\Gmp',
+    protected $aliases = [
+        'Bcmath' => 'Zend\Math\BigInteger\Adapter\Bcmath',
+        'Gmp'    => 'Zend\Math\BigInteger\Adapter\Gmp',
+    ];
```

Comment:
Actually, unless the code is using the TitleCased version, we *should* lowercase the aliases to be consistent with the original definitions.